### ISSUE TITLE
2.50.0 false positive UnrecognisedJavadocTag with Lombok

### ISSUE THREAD

**Issue by lbenedetto** [google/error-prone#5855]
2.50.0 does not accept this code.
```java
import lombok.Getter;

@Getter
public class Reproducer {
  /**
   * See {@link String}
   */
  private String str = "";
}
```

```
warning: [UnrecognisedJavadocTag] This Javadoc tag wasn't recognised by the parser. Is it malformed somehow, perhaps with mismatched braces?
  private String str = "";
  ^
```

Issue goes away if I get rid of Lombok `@Getter` annotation or switch to markdown javadoc syntax

**Comment by robvanmaris** [comment]
FYI: the same issue occurs with the Lombok annotation on field- instead of class level, e.g.:
```java
import lombok.Getter;

public class Reproducer {
  /** See {@link String} */
  @Getter private String str = "";
}
```
